client/express-checkout/blocks/index.js

```diff
@@ -56,9 +61,16 @@ const expressCheckoutElementGooglePay = ( api ) => {
 				expressPaymentMethod="googlePay"
 			/>
 		),
-		edit: <GooglePayPreview />,
+		edit: (
+			<ExpressCheckoutContainer
+				api={ api }
+				expressPaymentMethod="googlePay"
+				isPreview
+			/>
+		),
 		supports: {
 			features: getConfig( 'features' ),
+			style: [ 'height', 'borderRadius' ],
```

Comment:
I tried commenting this out and it didn't change anything for me. Isn't this for the new Express Checkout settings API? I'm not sure we actually support that yet, but I'll admit I haven't been following the work on that for a long time 🤔 

Maybe I need to install Gutenberg or something to see the difference?

Reply:
Just to confirm, did you got the latest WooCommerce `trunk` branch in your site as described in the 4th step of **Test: Ensure ECE buttons support styling controls in the Block editor**? It looks like it worked for the WooPay button in #9473. You can check https://github.com/woocommerce/woocommerce/pull/51296 to have an idea how it should look like.

If you can't test it for some reason, I think that's fine as this doesn't impact the main purposes of this PR, and the lack of support in the current WooCommerce version doesn't make it of an issue either. Honestly, I wasn't able to test myself because the WooCommerce build command was failing for me, but I figured it should work out of the box when the support comes to the stable WC version.

Reply:
Hmm ok ok, I'll see if I can test with WC `trunk` and se if that works!

Reply:
Built WC `trunk` and can confirm this works as expected 👍